packages/venia-ui/lib/components/Gallery/__tests__/__snapshots__/gallery.spec.js.snap

```diff
@@ -6,21 +6,33 @@ exports[`renders if \`items\` is an array of objects 1`] = `
 >
   <div>
     <div>
-      <img
-        alt=""
-        height="375"
-        src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAAFCAQAAADIpIVQAAAADklEQVR42mNkgAJGIhgAALQABsHyMOcAAAAASUVORK5CYII="
-        width="300"
-      />
-      <img
-        alt="Test Product 1"
-        height="375"
-        loading="lazy"
-        sizes="300px"
-        src="a.url"
-        srcSet="a.url 40w,a.url 80w,a.url 160w,a.url 320w,a.url 640w,a.url 1280w,a.url 1600w,a.url 2560w"
-        width="300"
-      />
+      <div
+        className="undefined undefined"
```

Comment:
I think you can correct this in snapshots by mocking classify ie `jest.mock('../../classify');`

Reply:
Thank you! Resolved in https://github.com/magento/pwa-studio/pull/1871/commits/8e815492172cac77557241c4e7dc51a15a634220.